configure.ac

```diff
@@ -39,10 +39,8 @@ dnl Checks for library functions.
 AC_FUNC_ALLOCA
 AC_FUNC_CHOWN
 AC_FUNC_FORK
-AC_FUNC_MALLOC
 AC_FUNC_MKTIME
 AC_FUNC_MMAP
-AC_FUNC_REALLOC
```

Comment:
Those two checks basically make sure that allocating 0 bytes yields a valid pointer.  I am not aware of any 0-byte allocations.  So I think we don't need them.  Otherwise we should use gnulibs `malloc-gnu` and `realloc-gnu` modules.

Reply:
> I am not aware of any 0-byte allocations

right. 
I would rather like that a malloc(0) fails with a NULL pointer, that way you spot such coding errors more easily, at least at runtime.